client/src/modules/depots/modals/depot.modal.html

```diff
@@ -153,20 +153,37 @@
 
     <div ng-if="DepotModalCtrl.enable_strict_depot_distribution">
       <label class="control-label" translate>DEPOT.ALLOWED_DESTINATION_DEPOTS</label>
-      <p ng-if="!DepotModalCtrl.depot.allowed_distribution_depots.length"><i translate>DEPOT.NO_DEPOT</i></p>
-      <bh-depot-search-select label="STOCK.DEPOT" id="distribution_depots"
-        depots-uuids="DepotModalCtrl.depot.allowed_distribution_depots"
-        on-change="DepotModalCtrl.onDistributionDepotChange(depots)"
-        required="DepotModalCtrl.enable_strict_depot_distribution">
-      </bh-depot-search-select>
+
+      <div class="row">
+        <div class="col-md-6">
+          <div ng-repeat ="r in DepotModalCtrl.depotRight">
+            <div class="checkbox">
+              <label>
+                <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
+                <span translate> {{ r.text }} </span>
+              </label>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div ng-repeat ="l in DepotModalCtrl.depotLeft">
+            <div class="checkbox">
+              <label>
+                <input type="checkbox" ng-model="l.checked"  ng-true-value="1" ng-false-value="0">
+                <span translate> {{ l.text }} </span>
+              </label>
+            </div>
+          </div>
+        </div>
```

Comment:
If depotLeft and depotRight are just for displaying depots left and right, you could use just one loop such as : 
```html
<div ng-repeat ="r in DepotModalCtrl.depots" class="col-md-6">
  <div class="checkbox">
    <label>
      <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
      <span translate> {{ r.text }} </span>
    </label>
  </div>
</div>
```

Reply:
![image](https://user-images.githubusercontent.com/8035588/120292967-b0c0d600-c2bc-11eb-862a-4a5c392b40ac.png)

I tried with this option, the display stays on one column, my idea on concerning the two-column display was just not to have a very long list

Reply:
Take a look at the answer to this:  https://stackoverflow.com/questions/25655207/angularjs-how-to-dynamically-split-list-in-multiple-columns